### PR TITLE
Added test for changeset reader

### PR DESCRIPTION
**Pull request by khanaffan**
Changeset reader let you enumerate changeset from javascript size for debugging and analysis purpose.

### CODE REVIEW

File: core/backend/src/test/standalone/ChangesetReader.test.ts [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+
+// spell-checker: disable
+
+describe("changeset reader", () => {
+  it("read data using getColumnValue()", () => {
+    const changesetFile = path.join(KnownTestLocations.assetsDir, "changesets/data.changeset");
+    const reader: IModelJsNative.ChangesetReader = new IModelHost.platform.ChangesetReader();
+    assert(reader.open(changesetFile, false) === DbResult.BE_SQLITE_OK);
+
+    assert(reader.step() === DbResult.BE_SQLITE_ROW);
```
Comment (pmconne) [inline core/backend/src/test/standalone/ChangesetReader.test.ts:19]: Why are using `assert` from bentleyjs-core instead of chai's `assert` or `expect`?

**Review by pmconne — CHANGES_REQUESTED** [review]

File: core/backend/src/test/standalone/ChangesetReader.test.ts (hunk 1 above)
Comment (khanaffan) [reply core/backend/src/test/standalone/ChangesetReader.test.ts:19]: Did not realized i was using that one. I fixed it.

**Review by pmconne — APPROVED** [review]

**Comment by pmconne** [comment]
@khanaffan I am setting auto-merge to prevent this PR from constantly rebuilding itself every time anyone else merges to master. If you don't want to merge it yet, please close it.

**Comment by pmconne** [comment]
Need to run rush change. Closing to stop it trying to rebuild every time another PR completes.

**Comment by calebmshafer** [comment]
@pmconne for future reference if you "Request changes", that will block mergify from updating the branch thus stopping the continuous build cycle.

**Comment by pmconne** [comment]
> @pmconne for future reference if you "Request changes", that will block mergify from updating the branch thus stopping the continuous build cycle.

@calebmshafer would it be possible to prevent rebuild on merge if auto-merge is not enabled? Lots of PRs just sit in a constant rebuild cycle waiting for someone to get around to manually merging them.

**Comment by calebmshafer** [comment]
@pmconne they do not support that currently, probably due to it being a new GitHub feature.  One thing we could do is not merge with master if the builds have not succeeded.  If they do succeed, then we'd merge.

This would prevent us continuously merging into a PR that is going to fail.

**Comment by pmconne** [comment]
@calebmshafer yeah, unfortunately that would probably mean 9 times out of 10 a flaky integration test kills your build.

**Comment by calebmshafer** [comment]
@pmconne we probably should just fix our flaky tests :)  Maybe we give a try and see?

I'll go through and disable all known flaky tests today if I can't fix them.

**Pull request closed without merging**